src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntityDueDiligenceValidatorTest.java

```diff
@@ -30,12 +30,20 @@ void init() {
     }
 
     @Test
-    void testErrorReportedWhenIdentityDateFieldIsNow() {
+    void testNoErrorReportedWhenIdentityDateFieldIsNow() {
         overseasEntityDueDiligenceDto.setIdentityDate(LocalDate.now());
         Errors errors = overseasEntityDueDiligenceValidator.validate(overseasEntityDueDiligenceDto, new Errors(), CONTEXT);
         assertFalse(errors.hasErrors());
     }
 
+    @Test
+    void testNoErrorReportedWhenCountryIsInTheUk() {
+        overseasEntityDueDiligenceDto.setIdentityDate(LocalDate.now());
+        overseasEntityDueDiligenceDto.getAddress().setCountry("England");
```

Comment:
i changed England to HELLO and the test passes. We'll have to validate the country is in the uk so perhaps have a separate list of uk countries, and in the addresDTO validator, rather than just skip the country check if on due diligence, have it call a new function to check if country is in the uk country list

Reply:
Ok, I'll look into that.